app/facades/quotable_facade.rb

```diff
@@ -0,0 +1,15 @@
+class QuotableFacade
+
+  def self.search_for_quotes(query)
+    # require "pry"; binding.pry
+    json = QuotableService.search_quotes(query)
+    if json[:results].count > 0
+      json[:results][0..9].map {|quote| Quote.new(quote, json[:count])}
+    else
+      [Quote.new({results: {content: [], tags: [], author: []}}, json[:count])]
+    end
```

Comment:
This is an interesting implementation. Were you finding that you were getting errors when no results were returned?